tensorboard/plugins/hparams/tf_hparams_sessions_pane/tf-hparams-sessions-pane.ts

```diff
@@ -196,16 +196,20 @@ class TfHparamsSessionsPane extends PolymerElement {
     action: string,
     tabName?: string
   ) => {
-    // If window['ga'] is defined, use it, otherwise any logging calls will be
-    // no-ops. window['ga'] is only defined in the hosted TensorBoard.
+    // If window['dataLayer'] is defined, use it, otherwise any logging calls
+    // will be no-ops. window['dataLayer'] is only defined in the hosted
+    // TensorBoard.
     // @ts-ignore
-    const analytics = window['ga'] || function () {};
+    const dataLayer = window['dataLayer'] || [];
     // @ts-ignore
-    analytics('send', {
-      hitType: 'event',
-      eventCategory: 'HParams',
-      eventAction: action,
-      eventLabel: tabName,
+
+    function gtag(...ignore: unknown[]) {
+      dataLayer.push(arguments);
```

Comment:
I do not understand how this works. ...ignore? How is arguments defined?

Reply:
`arguments` is a special keyword in JavaScript:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/arguments
But in this case it is not really important to know how it works. It's just important to know that this is how the GA documentation wants you to define the gtag() function. So I added a link to the documentation.

As for the "...ignore: unknown[]", this was some funky workaround to get the TypeScript compiler to allow me to call the gtag() function with arguments. I agree it's confusing and distracting so I just removed it and moved the gtag() definition into the `@ts-ignore` block.